packages/management/framework/bundle/config/pacakgeinstalls/addons-manager-pi.yaml

```diff
@@ -0,0 +1,33 @@
+#@ load("@ytt:yaml", "yaml")
+#@ load("@ytt:data", "data")
+
+---
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageInstall
+metadata:
+  name: tanzu-addons-manager
+  namespace: #@ data.values.namespaceForPackageInstallation
+  annotations:
+    kapp.k14s.io/change-rule.0: "upsert after upserting addons-manager-packageinstall/serviceaccount"
+    kapp.k14s.io/change-rule.1: "delete before deleting addons-manager-packageinstall/serviceaccount"
+spec:
+  serviceAccountName: tanzu-addons-manager-package-sa
+  packageRef:
+    refName: addons-manager.tanzu.vmware.com #! TODO: confirm the addons manager package refName in uTKG
+    versionSelection:
```

Comment:
Pin to specific version or specify a range.

Reply:
Should that be configured as part of values.yaml? Other pkgis don't have versions specified.

Reply:
@anujc25 @vuil - What are doing with versions? Should this be followed up separately?

Reply:
Yes. I am planning to follow up on this as separately. Especially to make sure we handle the upgrade scenarios correctly. Currently we can keep to select the latest version.

Reply:
Looks like this can be resolved and @anujc25 will follow up on it.